lib/eslint/eslint-helpers.js

```diff
@@ -881,9 +917,10 @@ function processOptions({
  * if cacheFile points to a file or looks like a file then in will just use that file
  * @param {string} cacheFile The name of file to be used to store the cache
  * @param {string} cwd Current working directory
+ * @param {string} [filenamePrefix="cache_"] The prefix to use for the cache file
  * @returns {string} the resolved path to the cache file
  */
-function getCacheFile(cacheFile, cwd) {
+function getCacheFile(cacheFile, cwd, filenamePrefix) {
```

Comment:
Because we're adding a third parameter here, can we make it an options object? I think that helps readability of the code.

```suggestion
function getCacheFile(cacheFile, cwd, { filenamePrefix = "_cache" }) {
```